apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -184,7 +180,12 @@ function wpcom_track_global_styles( $blog_id, $post, $updated ) {
  * @return bool Returns true if custom styles are in use.
  */
 function wpcom_global_styles_in_use() {
-	$user_cpt = WP_Theme_JSON_Resolver_Gutenberg::get_user_data_from_wp_global_styles( wp_get_theme() );
+	if ( class_exists( 'WP_Theme_JSON_Resolver' ) ) {
+		$user_cpt = WP_Theme_JSON_Resolver::get_user_data_from_wp_global_styles( null );
+	} else {
+		do_action( 'global_styles_log', 'global_styles_not_in_use' );
```

Comment:
Does this mean that the site is running an old version of WP or that they're not using GS? Just wondering if it's worth a comment similarly to `wpcom_block_global_styles_frontend`

Reply:
I think you're right. We cannot tell for sure the site isn't using GS so let's not make any assumption :)